src/game.cpp

```diff
@@ -524,7 +524,7 @@ void game::init_ui( const bool resized )
     // Only refresh if we are in-game, otherwise all resources are not initialized
     // and this refresh will crash the game.
     if( resized && u.getID() != -1 ) {
-        refresh_all();
+        g->refresh_all();
```

Comment:
Simply `refresh_all()` whenever inside the `game` class.

(Here an all over the PR).

Reply:
Thanks. I wanted to make it consistent with other occurrences, but I guess I can do it later and actually by removing `g->` inside of `game` class.